src/LiveComponent/src/ComponentWithFormTrait.php

```diff
@@ -126,9 +126,7 @@ public function getFormName(): string
 
     public function getFormValues(): array
     {
-        if (null === $this->formValues) {
-            $this->formValues = $this->extractFormValues($this->getForm());
-        }
+        $this->formValues = $this->extractFormValues($this->getForm());
```

Comment:
If you were able to put together a failing test, that would be *awesome*. As you know, this change is also in #221 (with a slight difference). But, at the moment, even in that PR, there are no tests covering the problem (i.e. if I revert that change in #221, its tests still pass). So, a test to show off the problem would be 💯

Reply:
@Lustmored PR covers more cases than mine, so probably It'd be better to leave to put that test in it's PR, so closing this one :)